suite-native/module-dev-utils/src/screens/DemoScreen.tsx

```diff
@@ -1,38 +1,39 @@
 import { useRef, useState } from 'react';
 import { TextInput, View } from 'react-native';
 
-import { Link } from '@suite-native/link';
 import {
-    Text,
+    // Card,
+    // ListItemSkeleton,
+    AlertBox,
+    Badge,
+    BadgeVariant,
     Box,
-    Hint,
-    SearchInput,
-    Radio,
-    CheckBox,
-    Switch,
-    IconButton,
-    InputWrapper,
-    Input,
-    VStack,
     Button,
     ButtonColorScheme,
+    ButtonSize,
+    CheckBox,
     Divider,
-    Badge,
-    BadgeVariant,
+    Hint,
     HStack,
-    ButtonSize,
-    TextButton,
+    IconButton,
+    Input,
+    InputWrapper,
     NumPadButton,
+    Radio,
+    SearchInput,
+    Switch,
+    Text,
+    TextButton,
     TextButtonVariant,
-    // Card,
-    // ListItemSkeleton,
-    AlertBox,
+    VStack,
 } from '@suite-native/atoms';
-import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { Screen, ScreenSubHeader } from '@suite-native/navigation';
-import { CryptoIcon, Icon } from '@suite-native/icons';
 import { isDevelopOrDebugEnv } from '@suite-native/config';
+import { CryptoIcon, Icon } from '@suite-native/icons';
+import { Link } from '@suite-native/link';
+import { Screen, ScreenSubHeader } from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { TypographyStyle } from '@trezor/theme';
+import { UpdateProgressIndicatorDemo } from '@suite-native/firmware';
```

Comment:
Would it make sense to move it to this dev package?

Reply:
I am not sure, it's useful if you are able to place this component on Homepage for example, because it's faster then go to dev menu after reload. It could be import from dev package too but I don't want encourage people doing any import from `module-*`.